android/src/main/java/ie/gov/tracing/Tracing.kt

```diff
@@ -679,19 +698,19 @@ object Tracing {
             if (isPaused) {
                 exposureDisabledReason = "paused"
             }
+
             try {
-                // check bluetooth
-                val bluetoothAdapter: BluetoothAdapter? = BluetoothAdapter.getDefaultAdapter();
-                if (bluetoothAdapter != null && !bluetoothAdapter.isEnabled) {
+                if (!isBluetoothAvailable()) {
                     exposureStatus = EXPOSURE_STATUS_DISABLED
-                    exposureDisabledReason = "bluetooth";
+                    exposureDisabledReason = "bluetooth"
+                } else if (!isPaused) {
+                    if (exposureStatus == EXPOSURE_STATUS_ACTIVE) {
+                        exposureDisabledReason = ""
```

Comment:
Likewise I'm not sure if an empty string is the best choice for `exposureDisabledReason` if ENS is active but it seemed to make sense based on existing logic.

Reply:
empty string is the correct value